uint/fuzz/Cargo.toml

```
@@ -2,7 +2,7 @@
 name = "uint-fuzz"
 description = "Fuzzers for uint algorithms"
 publish = false
-version = "0.1.0"
+version = "0.1.1"
```

no need for a bump (not published)
```suggestion
version = "0.1.0"
```